ncm-metaconfig/src/main/metaconfig/kafka/pan/schema.pan

```diff
@@ -2,26 +2,198 @@ declaration template metaconfig/kafka/schema;
 
 include 'pan/types';
 
-
 type kafka_server_properties = {
+    "zookeeper.connect" : string
+    "advertised.host.name" ? string
     "advertised.listeners" ? string = "PLAINTEXT://localhost:9092"
+    "advertised.port" ? long(0..)
+    "auto.create.topics.enable" ? choice("true", "false")
+    "auto.leader.rebalance.enable" ? choice("true", "false")
+    "background.threads" ? long(1..)
     "broker.id" : long(0..)
-    "group.initial.rebalance.delay.ms" ? long(0..)
-    "inter.broker.listener.name" ? string
-    "listener.security.protocol.map" ? string
+    "compression.type" ? string
```

Comment:
Is there a list of valid compression types that could be supplied here? (Sorry I don't know that much about Kafka)